spec/features/password_length_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+require 'simple_send_keys'
+
+RSpec.feature "new user password", type: :feature, js: true do
+  scenario "user cannot input password more than 10 characters" do
+    visit('/')
```

Comment:
Can we use the short form for 'user sign up' from test helper? just to keep the code DRY

Reply:
Sorry my bad, we can't! :D

Reply:
Yeah unfortunately not in this case, because here we want them to be _not signed up/in___